src/models/GitWalkerFs.js

```diff
@@ -115,6 +117,7 @@ export class GitWalkerFs {
 
   async oid(entry) {
     if (entry._oid === false) {
+      const me = this
```

Comment:
I would use a common `self` or `that` instead of `me`.

Reply:
done